feature/migration/qrcode/build.gradle.kts

```diff
@@ -10,6 +9,7 @@ android {
 
 dependencies {
     implementation(projects.core.common)
+    implementation(projects.legacy.ui.base)
 
     implementation(projects.core.ui.compose.designsystem)
     debugImplementation(projects.core.ui.compose.theme2.k9mail)
```

Comment:
This should not be necessary as it's part of the debug api of the `designsystem` module.